llvm/include/llvm/CodeGen/GlobalISel/CombinerHelper.h

```diff
@@ -853,6 +853,9 @@ class CombinerHelper {
   bool matchExtractVectorElementWithDifferentIndices(const MachineOperand &MO,
                                                      BuildFnTy &MatchInfo);
 
+  /// Combine insert vector element OOB.
+  bool matchInsertVectorElementOOB(MachineInstr &MI, BuildFnTy &MatchInfo);
```

Comment:
Please clarify what OOB means, it is not obvious.

Reply:
out of bound